app/views/layouts/application.html.haml

```diff
@@ -0,0 +1,22 @@
+!!!
+%html
+  %head
+    %meta(charset="UTF-8")
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))